profiles/spectrumscale-ece/tuned.conf

```diff
@@ -6,7 +6,29 @@
 summary=Optimized for Spectrum Scale Erasure Code Edition Servers
 include=throughput-performance
 
+[cpu]
+governor=performance
+energy_perf_bias=performance
+min_perf_pct=100
+
 [sysctl]
+kernel.sched_min_granularity_ns = 10000000
+kernel.sched_wakeup_granularity_ns = 15000000
+kernel.numa_balancing = 1
+vm.dirty_ratio = 40
+vm.dirty_background_ratio = 10
+vm.swappiness=10
 net.ipv4.tcp_window_scaling = 1
 net.ipv4.tcp_timestamps = 1
-kernel.numa_balancing = 1
+
+[disk-sas]
+type=disk
+devices = sd*
+elevator = mq-deadline
+readahead = 0
+
+[disk-nvme]
+type=disk
+devices = nvme*
```

Comment:
Does the tuning really get applied to these disks? I remember there was a problem with Tuned not recognizing nvme disks.

Reply:
Well on RH8 tests it does at least in our lab

Reply:
OK. I just wanted to ask.

Reply:
Sure. We have not tested on Fedora but the goal of this profile is not to be run in Fedora as ECE itself is not supported on Fedora. We been running with this parameters for over a month on many test clusters and as soon we have this published we will push for the RH move.